tests/publiccloud/run_ltp.pm

```diff
@@ -47,21 +45,20 @@ sub instance_log_args
 
 sub upload_ltp_logs
 {
-    my ($self) = @_;
     record_info('LTP Logs', 'upload');
-    assert_script_run("test -f $root_dir/result.json || echo No result log");
-    parse_extra_log('LTP', "$root_dir/result.json");
-    # debug file in the standart LTP log-dir. structure:
-    assert_script_run("test -f /tmp/runltp.\$USER/latest/debug.log || echo No debug log");
-    upload_logs("/tmp/runltp.\$USER/latest/debug.log", failok => 1);
+    assert_script_run("test -f /tmp/kirk.\$USER/latest/debug.log || echo No debug log");
```

Comment:
but due to use of `||` this command will never fail so what the point to use `assert_script_run` here ? 
also another problem that no matter if file exists or not we will try to upload it in next line :thinking: 

what if instead we will do something like : 

```
if ( script_run("test -f /tmp/kirk.\$USER/latest/debug.log")) {
upload_logs("/tmp/kirk.\$USER/latest/debug.log", log_name => 'debug.log.txt', failok => 1);
}
else {
 record_info("debug.log", "/tmp/kirk.\$USER/latest/debug.log does not exists", result => "fail");
}
```